smithy-aws-protocol-tests/model/restXml/document-maps.smithy

```diff
@@ -101,7 +101,6 @@ apply XmlMaps @httpResponseTests([
 ])
 
 @http(uri: "/XmlEmptyMaps", method: "POST")
-@tags(["client-only"])
 operation XmlEmptyMaps {
```

Comment:
Looks like there's a set of `@httpResponseTests` for this operation. Should they have the `appliesTo` member added?